mantidproject/mantid
# Simplify sphere and ellipsoid glyphs in the VSI

The ellipsoidal glyph in the splatter plot view overwhelms the actual peak contained inside. I toned this down by just showing the axes of the ellipsoid around each peak.

**To test:**



code review: 
Verify that `vtkPeakMarkerFactory::getTransformTensor` calculates that correct tranformation for [vtkTensorGlyph](http://www.vtk.org/doc/nightly/html/classvtkTensorGlyph.html). I found [this example](http://www.vtk.org/Wiki/VTK/Examples/Cxx/Visualization/TensorGlyph) useful for understanding how to use this filter.

Double check that the compiler-generated copy constructor, copy assignment operator and destructor are sufficient. 

spheres: 
Run the [IntegratePeaksMD](http://docs.mantidproject.org/nightly/algorithms/IntegratePeaksMD-v2.html) example and view `TOPAZ_3132_md` and `peaks` in the VSI. Verify that the spheres are visible and the shape matches previous results.

ellipsoids:
Run the [IntegrateEllipsoids](http://docs.mantidproject.org/nightly/algorithms/IntegrateEllipsoids-v1.html) example and view `TOPAZ_3132_md` and `TOPAZ_3132_peaks` in the VSI. Verify that the ellipses are visible and the shape matches previous results.

Fixes #15239.



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
